src/Codeception/Subscriber/BeforeAfterTest.php

```diff
@@ -30,16 +33,31 @@ class BeforeAfterTest implements EventSubscriberInterface
 
     protected array $startedTests = [];
 
+    private ?Actor $actor = null;
+
+    private ?Scenario $scenario = null;
+
     public function beforeClass(SuiteEvent $event): void
     {
         foreach ($event->getSuite()->getTests() as $test) {
-            $testClass = $test::class;
+            if ($test instanceof Cest) {
+                $testClass = $test->getTestClass()::class;
```

Comment:
`getTestClass()::class`
https://github.com/Codeception/Codeception/blob/522c7f0f1be5b700ffb8ef8342ba6595f4e16242/src/Codeception/Test/Cest.php#L230-L233
Could we consider renaming that method?

Reply:
Renamed